src/styles/_mixins.scss

```diff
@@ -58,3 +58,7 @@ $shadow-color: rgba(0, 0, 0, 0.2);
   right: 2rem;
   z-index: 1;
 }
+
+@mixin focused-element {
```

Comment:
you can try:
```css
@mixin focusable {
  &:focus-visible {
    box-shadow: ...
  }
}
```
This way, you just add the mixin to any element that can have focus:
```css
button {
  @include focusable;
}
```